en/developer_guide/logs.rst

```diff
@@ -0,0 +1,90 @@
+Logs
+====
+
+Our backoffice logging is based on Monolog.
+We provide an interface to display all the user action on the website
+It displays the date and time, user's IP, user's name, the current site's name and the log's message.
+
+Configuration
+-------------
+
+Monolog uses channels to record the log. To link a channel and a record,you will have to define an handler.
+The configuration of the handler can be placed in the app/config/log.yml file.
+
+To save the logs in MongoDB, We need to define a handler
+with connection's information of the base and collection's name.
+Level indicates information's type to record in the logs.
+For example 200 it's for info but there is also debug, error, etc, ...
+
+.. code-block:: yaml
+
+        mongo:
+            type: mongo
+            level: 200
+            mongo:
+                host: "localhost"
+                database: "%open_orchestra_cms.mongodb.database%"
+                collection: log
+            channels: [openorchestra]
+
+
+Channels defined the channels where are written the logs.
+If no channel is indicated then the logs are written in all channels.
+It's possible to put multiple channels or exclude one.
+The services who have the channel ``openorchestra`` write only in this chanel
+and our handler shows logs who are in the ``openorchestra`` channel.
+
+Processor
+---------
+
+The loggers who that allow record logs have the date and time but not IP address and user name.
+For save IP address and user name, we have created a processor.
+In the processor tags, we spent the channel ``openorchestra``.
+So, for each log written in this channel he spends into the processor.
+
+The processor must be tagged by:
+
+.. code-block:: yaml
+
+    tags:
+        - { name: monolog.processor, method: processRecord, channel: openorchestra }
+
+Logger
+------
+
+Services that write the logs are event subscriber. They must have in parameter ``Symfony\Bridge\Monolog\Logger``
+and be tagged by ``monolog.logger``
+
+For instance :
+
+.. code-block:: yaml
+
+    open_orchestra_log.subscriber.node:
+        class: %open_orchestra_log.subscriber.node.class%
+        arguments: [@logger]
+        tags:
+            - { name: kernel.event_subscriber }
+            - { name: monolog.logger, channel: openorchestra }
+
+On Open Orchestra when an action like updating a node is done an event is created.
```

Comment:
"In Open Orchestra an event is dispatched on most user actions (ie. creation, update of elements...)."